src/django/api/facility_history.py

```diff
@@ -4,6 +4,54 @@
 from api.models import FacilityMatch
 
 
+def create_associate_match_string(facility_str, contributor_str, list_str):
+    return 'Associate facility {} with contributor {} via list {}'.format(
+        facility_str,
+        contributor_str,
+        list_str,
+    )
+
+
+def create_associate_match_change_reason(list_item, facility):
```

Comment:
This is a separate function because it appears that the change reason field is VARCHAR(100), which makes it impossible to serialize every list name. In practice, the changes here don't depend on the change_reason existing at all, so I'm happy to dump writing to changeReason at all.